cmd/containerd-shim-runhcs-v1/delete.go

```diff
@@ -123,6 +127,30 @@ The delete command will be executed in the container's bundle as its cwd.
 			fmt.Fprintf(os.Stderr, "failed to delete user %q: %v", username, err)
 		}
 
+		// cleanup the layers mounted for the container. We currently only handle cleanup of CimFS
+		// layers here.  First n-1 values should be the image layerFolders (topmost layer being at
+		// index 0) and the last entry should be the scratch layer
+		var layerFolders []string
+		f, err := os.Open(filepath.Join(bundleFlag, layersFile))
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "open layers file: %s", err)
+		} else {
+			defer f.Close()
+			if err = json.NewDecoder(f).Decode(&layerFolders); err != nil {
+				fmt.Fprintf(os.Stderr, "decode layers json: %s", err)
+			}
+		}
+		if err == nil && cimlayer.IsCimLayer(layerFolders[0]) {
+			scratchLayerFolderPath := layerFolders[len(layerFolders)-1]
```

Comment:
Should this be the same logic as in layer closer Release()?

Reply:
I updated the code to make it a common function.